website/src/pages/showcase/_components/ShowcaseFilterToggle/index.tsx

```diff
@@ -55,7 +55,6 @@ export default function ShowcaseFilterToggle(): JSX.Element {
       <label htmlFor={id} className={clsx(styles.checkboxLabel, 'shadow--md')}>
```

Comment:
I added `shadow--md` here but it looks like it's also in the `. checkboxLabel` class so it's duplicated